__tests__/endpoints.test.js

```diff
@@ -2,10 +2,70 @@ const request = require("supertest");
 const app = require("../app/app");
 const endpoints = require("../endpoints.json");
 
-describe("/api", () => {
-  test("GET /api - success", async () => {
-    const response = await request(app).get("/api");
-    expect(response.status).toBe(200);
-    expect(response.body).toEqual(endpoints);
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed.js");
+const db = require("../db/connection.js");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("endpoints", () => {
+  describe("/api", () => {
+    test("GET /api - success", async () => {
+      const response = await request(app).get("/api");
+      expect(response.status).toBe(200);
+      expect(response.body).toEqual(endpoints);
+    });
+  });
+
+  describe("/api/topics", () => {
+    test("GET /api/topics - success", async () => {
+      const response = await request(app).get("/api/topics");
+      expect(response.status).toBe(200);
+      expect(response.body.topics).toBeInstanceOf(Array);
+      expect(response.body.topics).toHaveLength(3);
+      response.body.topics.forEach((topic) => {
+        expect(topic).toEqual(
+          expect.objectContaining({
+            slug: expect.any(String),
+            description: expect.any(String),
+          })
+        );
+      });
+    });
+  });
+
+  describe("/api/articles/:article_id", () => {
+    test("GET /api/articles/:article_id - success", async () => {
+      const response = await request(app).get("/api/articles/1");
+      expect(response.status).toBe(200);
+      expect(response.body.article).toHaveProperty("author");
+      expect(response.body.article).toHaveProperty("title");
+      expect(response.body.article).toHaveProperty("article_id");
+      expect(response.body.article).toHaveProperty("body");
+      expect(response.body.article).toHaveProperty("topic");
+      expect(response.body.article).toHaveProperty("created_at");
+      expect(response.body.article).toHaveProperty("votes");
+      expect(response.body.article).toHaveProperty("article_img_url");
```

Comment:
Since we are looking at 1 specific article, we have the information as to what author, title, id etc. we are expecting specifically so we should be more specific in our test! :-)